libraries/forms.py

```diff
@@ -456,9 +474,12 @@ def _generate_hyperkitty_word_cloud(self, version):
         word_frequencies = {}
         for content in self._get_mail_content(version):
             for key, val in wc.process_text(content).items():
-                if key not in word_frequencies:
-                    word_frequencies[key] = 0
-                word_frequencies[key] += val
+                if len(key) < 2:
+                    continue
+                keyl = key.lower()
+                if keyl not in word_frequencies:
+                    word_frequencies[keyl] = 0
+                word_frequencies[keyl] += val
```

Comment:
I thought this was `key1` for a while. Can we be a bit more verbose but readable?

```suggestion
                key_lower = key.lower()
                if key_lower not in word_frequencies:
                    word_frequencies[key_lower] = 0
                word_frequencies[key_lower] += val
```